.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Generate changelog
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3
+        with:
+          github-token: ${{ secrets.github_token }}
+          output-file: "false"
+      - name: Install deps
+        run: yarn
+      - name: Build App
+        env:
+          RELEASE_VERSION: ${{ steps.changelog.outputs.tag }}
+        run: |
+          yarn build
+          tar czf build-${RELEASE_VERSION}.tar.gz ./build
```

Comment:
Let's add `ott-web-app-build-` to the names here

Reply:
Done